### ISSUE TITLE
Add inverse hyperbolic functions

### ISSUE THREAD

**Issue by infusion** [mathjax/MathJax#1714]
Hi,

it would be great, if you could add the following inverse hyperbolic functions to the known symbol scope:

- \asinh
- \acosh
- \atanh
- \acoth

Thanks!

**Comment by pkra** [comment]
Thanks for filing this feature request. Note that you can define your own macros, cf. http://docs.mathjax.org/en/latest/tex.html#defining-tex-macros

**Comment by infusion** [comment]
Yep, that's the way I have it now. However, I generate a document out of pandoc and have to manually edit the generated html file for this extension. This doesn't hurt much, but adding these 4 symbols could maybe complete the list.
